src/renderer/windows/main.js

```diff
@@ -4,8 +4,28 @@ import ReactDOM from 'react-dom';
 import injectTapEventPlugin from 'react-tap-event-plugin';
 
 import PlayerPage from '../ui/pages/PlayerPage';
+import SettingsClass from '../../main/utils/Settings';
 
 injectTapEventPlugin();
 
+// Instantiate a settings object for checking 'start minimized'
```

Comment:
Oh dang - that's my fault. I wasn't sure about how `global` worked. This block and line 7 aren't necessary then.